<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="breakfast_menu">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>500</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="breakfast_star_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../kakaotalk_bot/dev/star.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="breakfast_star_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>150</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../kakaotalk_bot/dev/star.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="breakfast_star_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="breakfast_star_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="breakfast_star_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>150</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../kakaotalk_bot/dev/star.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lunch_star_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lunch_star_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lunch_star_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="lunch_menu">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>500</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lunch_star_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lunch_star_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="dinner_menu">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>500</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="dinner_star_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dinner_star_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>490</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dinner_star_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>490</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dinner_star_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>490</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="dinner_star_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>490</y>
     <width>39</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="send_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>600</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="breakfast_text">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="lunch_text">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="dinner_text">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>390</y>
     <width>300</width>
     <height>70</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
